ptlshs/conn.go

```diff
@@ -317,6 +321,10 @@ func serverConnWithCache(toClient net.Conn, cfg ListenerConfig, cache *nonceCach
 		newOnce(), newOnce(), newDeadline(), newDeadline(), sync.Mutex{}}
 }
 
+func (c *serverConn) Wrapped() net.Conn {
+	return c.wrapped
```

Comment:
Similar to the `clientConn` case, this needs locking.  In this case, there's an existing lock you can use: `c.wrappedLock`.

Reply:
Fixed.